ADBench/run-all.ps1

```diff
@@ -119,6 +172,7 @@ function run_command ($indent, $outfile, $timeout, $cmd) {
     } else {
         $Process.Kill()
         Write-Host "${indent}Killed after $timeout seconds"
+        Store-NonFatalError "Process killed after $timeout seconds`n[$cmd $args]"
```

Comment:
Might it be better to use `Report-NonFatalError` here and get rid of the `Write-Host` above?

Reply:
I don't think so. Non-fatal errors are printed again at the end of the script, so that the user wouldn't have to read through the whole log to check if they occurred. `"${indent}Killed after $timeout seconds"` won't be a useful error message there, because it lacks necessary context (what was killed) by itself. So I put this context into the stored message. However when we kill the process, all the context was just printed into the terminal and it won't make much sense to repeat it. Hence the different texts to print when the error just occurred and in the end.

Reply:
OK, I don't really mind.  I prefer less code to more but as you wish.